### PR TITLE
MUMMNG-2404 Turning off old my.wisc

### PR DESCRIPTION
**Pull request by vertein**
We are encouraging uw-frame apps to no longer show the back to MyUW old in the footer.

### CODE REVIEW

File: uw-frame-components/static.html [hunk 1]
```diff
@@ -29,13 +29,6 @@
         margin: -8px;
         font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;"
       id="loading-splash">
-        <!--Bad browser error message-->
-        <!--[if lt IE 10]>
-        <div class="bad-browser">
-          <span class="fa fa-frown-o"></span>
-          <p>Sorry, MyUW beta does not support your browser.<br/><a href="https://kb.wisc.edu/myuw/page.php?id=51345">Learn how to upgrade your browser.</a><br/>Can't upgrade? <a href="http://my.wisc.edu/portal/Login?profile=default">Switch back to MyUW classic.</a></p>
-        </div>
-        <![endif]-->
```
Comment (timlevett) [inline uw-frame-components/static.html]: We still want this error message, just remove the "go back to classic MyUW" part.

Comment (vertein) [inline uw-frame-components/static.html]: It was commented out.  Was it meant to be uncommented?

Comment (timlevett) [inline uw-frame-components/static.html]: That is the way these work. Knowledge drop: https://msdn.microsoft.com/en-us/library/ms537512(v=vs.85).aspx and http://www.quirksmode.org/css/condcom.html

Comment (vertein) [inline uw-frame-components/static.html]: Sounds good.  Just wanted to make sure that was done on purpose.  Changed back.

**Comment by timlevett** [comment]
Odd this is a MUMMNG but no blocker. :+1:

**Comment by vertein** [comment]
I didn't even think about it.  I'll change our jira key over.

**Pull request merged**